bin/cli.js

```diff
@@ -68,14 +68,17 @@ if(pathArg === 'demo') {
     }
 }
 
-theme = glob.sync('css/theme/*.css', {
-    cwd: revealPath
-}).concat(glob.sync('theme/*.css', {
-    cwd: path.resolve(basePath)
-})).filter(function(themePath) {
-    return path.basename(themePath).replace(path.extname(themePath), '') === program.theme;
-}).pop() || 'css/theme/' + theme + '.css';
-
+if (program.theme.startsWith("http") && program.theme.endsWith(".css")) {
```

Comment:
Not sure in what version of Node/V8 `starts/endsWith` was introduced, but perhaps it would be easy to still support older versions?